### PR TITLE
Ensure that "about" and other static pages all have title tags

### PR DESCRIPTION
**Pull request by lollypop27**
Add SEO tags to static pages

### CODE REVIEW

**Comment by lollypop27** [comment]
@tansaku ready for review

File: app/controllers/organisations_controller.rb [hunk 1]
```diff
@@ -120,6 +120,10 @@ def self.build params
         category_ids: []
       )
     end
+
+    def entity
```
Comment (tansaku) [inline app/controllers/organisations_controller.rb]: I don't think we need these entity methods any more do we?

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ def new
 
   # GET /pages/:permalink/edit
   def edit
-    @page = Page.find_by_permalink!(params[:id])
+
```
Comment (tansaku) [inline app/controllers/pages_controller.rb]: can remove this empty line

File: app/controllers/volunteer_ops_controller.rb [hunk 3]
```diff
@@ -63,6 +63,10 @@ def volunteer_op_params
     params.require(:volunteer_op).permit(*args)
   end
 
+  def entity
```
Comment (tansaku) [inline app/controllers/volunteer_ops_controller.rb]: I don't think we need these entity methods any more do we?

File: features/admin/seo.feature [hunk 4]
```diff
@@ -23,3 +26,9 @@ Feature: Would like the site to appear high in Google's listing
   Scenario: Meta Title and Description for volunteers show page
     Given I visit the show page for the volunteer_op titled "Helping Volunteer"
     Then I should have a page with a title: "Helping Volunteer | Harrow volunteering"
+
+  Scenario: Meta Title and Description for about us show page
+    Given I am on the show page with the "aboutus" permalink
+#    And I debug
```
Comment (tansaku) [inline features/admin/seo.feature]: let's remove this comment

File: features/admin/seo.feature [hunk 5]
```diff
@@ -10,6 +10,9 @@ Feature: Would like the site to appear high in Google's listing
     And the following volunteer opportunities exist:
       | title             | description           | organisation |
       | Helping Volunteer | Helping the earlderly | Friendly     |
+    And the following page exist:
```
Comment (tansaku) [inline features/admin/seo.feature]: might be nice to fix the grammar here

**Comment by tansaku** [comment]
thanks @lollypop27 - I've left a few suggested changes - see what you think ...

File: app/controllers/organisations_controller.rb (hunk 1 above)
Comment (lollypop27) [inline app/controllers/organisations_controller.rb]: @tansaku correct will remove

File: app/views/pages/aboutus.html.erb [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<strong><h2>About Us</h2></strong>
+<strong><h2><%= @page.name 'About Us' %></h2></strong>
```
Comment (lollypop27) [inline app/views/pages/aboutus.html.erb]: @tansaku It needs to be there or shows blank, if we take it out

Comment (tansaku) [inline app/views/pages/aboutus.html.erb]: that seems very strange.  what is `@page.name` doing then?  perhaps we shouldn't have `@page.name` there then?

**Comment by lollypop27** [comment]
@tansaku ready for review

**Comment by johnnymo87** [comment]
I'd prefer it if the data fetching/transforming part of the `set_tags` logic was extracted from the controllers, but that's not relevant to this PR. +1

File: app/views/pages/aboutus.html.erb (hunk 6 above)
Comment (tansaku) [inline app/views/pages/aboutus.html.erb]: I think we need more investigation here before we can merge this - seems like strange duplication

**Comment by lollypop27** [comment]
@tansaku ready for review

**Comment by tansaku** [comment]
@lollypop27 - looking good!

@johnnymo87 - want to create a refactoring ticket for that issue you mention for `set_tags`  maybe with some suggestions for techniques?

I wonder if I should create a refactoring ticket for removing/reviewing the controller specs ...

**Comment by tansaku** [comment]
@johnnymo87 I created a refactoring ticket for you https://www.pivotaltracker.com/story/show/135509695 and for me https://www.pivotaltracker.com/story/show/135510095

**Pull request merged**